src/main/java/xyz/aadev/generitech/common/tileentities/power/DistributePowerToFace.java

```diff
@@ -25,6 +25,7 @@
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.World;
 import net.minecraftforge.common.capabilities.Capability;
+import xyz.aadev.aalib.common.util.TileHelper;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'xyz.aadev.aalib.common.util.TileHelper'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUselessImportCheck)